vuepress/docs/v6.3.2/docs/curate/ecr-bundle-details.md

```diff
@@ -144,47 +144,47 @@ If you are using the `deploymentBaseName` property and want to install more vers
 
 ### Permissions
 
-The `permissions` property specifies a list of coupled clientIds and roles that will be bound in Keycloak. To find them, open the Keycloak console and navigate to _clients_ → _awesomeplugin-server_ → _Service Account Roles_.
+The `permissions` property specifies a list of coupled `clientIds` and roles that will be bound in Keycloak. To find them, open the Keycloak console and navigate to `clients` → `awesomeplugin-server` → `Service Account Roles`.
```

Comment:
I am wondering if we are supposed to call this Keycloak console or Entando Identity Management console? I've been confused about this while working on the import Keycloak tutorial. Is the 'awesomeplugin-server' real?

Reply:
1. awesomeplugin-server is made up, just some engineer getting cute.
2. I can't decide if it's better to use Keycloak console. I think it's a little less confusing in general, personally.

Reply:
colorful naming conventions bring joy to an otherwise dreary console... should we leave the awesomeness or use another proxy?

Reply:
We can keep it. I think it can be confusing when we invent things everywhere but there's no pattern here so no need to rework it.